addon/components/o-s-s/layout/sidebar.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="oss-sidebar--containers fx-col" ...attributes>
+  <div class="logo-container">
+    <a href="/">
+      <img src={{@logo}} alt="brand" />
+    </a>
```

Comment:
maybe replace it with something like `@goToRoot` argument which can be passed from outside. Because if we are on the identity app, clicking the logo needs to redirect to upfluence-web's index and not identity-web's.

cc @Miexil this might affect what you did in identity-web so you will need to pass `goToRoot` here